pgm-1.txt

```diff
@@ -0,0 +1,18 @@
+#! /bin/sh
+echo "Enter a pid:"
+read pid
+min=1
+max=32768
+if [ $pid -ge $min -a $pid -le $max ]
+	then
+		if ps -p $pid > /dev/null 2>&1
```

Comment:
'>' is for redirect standard output and '> /dev/null' is to ensure that any standard o/p from the execution of 'ps -p $pid' command gets redirected to /dev/null instead of appearing on the console during script execution.